cfpb/cfgov-refresh
# Add alternate "View more" URL field to Related Posts block

We have a page owner that would like to be able to use a different destination for the "View more" link on their Related Posts block, to go to the Blog, rather than the default of the Activity Log. This PR adds a new field to override that destination URL.

Background: https://GHE/CFGOV/platform/issues/2714

## Additions

- `alternate_view_more_url` field added to `RelatedPost`
- Related Post `_view_more()` macro updated to check for a value in the new field

## Testing

1. Pull branch
1. Run the migration
1. Edit/add a Related Posts block, adding something like `/about-us/blog/?categories=info-for-consumers` to the new field
1. Preview the page and see that the "View more" link is going to that URL, not the Activity Log

## Screenshots

![image](https://user-images.githubusercontent.com/1044670/47577913-0dde9900-d916-11e8-9094-2446b066271d.png)

## Checklist
